### PR TITLE
Added packages that chapter voting used

### PR DESCRIPTION
**Pull request by walkeronwater**

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -16,6 +16,9 @@
 \usepackage{verbatim}
 \usepackage{float}
 \usepackage{tikz}
+\usepackage{algpseudocode}
```
Comment (ezgimez) [inline main.tex]: oh, is this the substitute for:

`\usepackage[ruled, vlined]{algorithm2e}`

Comment (walkeronwater) [reply main.tex]: Yes, I can change the format if `\usepackage[ruled, vlined]{algorithm2e}` is our preference

Comment (ezgimez) [reply main.tex]: ? @walkeronwater  (let me tag you in case you didn't see it)

Comment (ezgimez) [reply main.tex]: oh ok
we are not writing the pseudocodes yet, so we can wait till we decide on the template maybe?

Comment (walkeronwater) [reply main.tex]: Yeah definitely fine. I'll do the change if needed

Comment (ezgimez) [reply main.tex]: :) thanks !

Comment (ezgimez) [reply main.tex]: so, can we remove this line for now? 
Line 19 --> to make sure that we don't face any package conflicts or stuff like that.

sorry for the back-and-forth communication

Comment (walkeronwater) [reply main.tex]: removed the packages (sorry for the delay)

**Review by ezgimez — APPROVED** [review]
lgtm

**Pull request merged**